examples/mobile/src/lib.rs

```diff
@@ -161,3 +165,18 @@ fn setup_music(asset_server: Res<AssetServer>, mut commands: Commands) {
         settings: PlaybackSettings::LOOP,
     });
 }
+
+// Pause audio when app goes into background and resume when it returns.
+// This is handled by the OS on iOS, but not on Android.
+fn handle_lifetime(
+    mut lifetime_events: EventReader<Lifetime>,
+    music_controller: Query<&AudioSink>,
+) {
+    for event in lifetime_events.read() {
+        match event {
+            Lifetime::Suspended => music_controller.single().pause(),
+            Lifetime::Resumed => music_controller.single().play(),
```

Comment:
One question- if we have AudioSink that was paused due to some other reasons wouldn't start playing as well "for no reason" from player/game maker perspective?

Reply:
Yeah, I agree with this analysis. We should probably cache the list of `Entity` that were paused here in a `Local<Vec<Entity>>`, in order to avoid this problem.

Reply:
this is an example. in real game code, people will have to handle that logic however they want, and not just for audio. switching the state to a pause menu for example